src/org/jitsi/util/RTPUtils.java

```diff
@@ -38,6 +38,22 @@ else if (diff > 1<<15)
     }
 
     /**
+     * Returns the difference between two RTP timestamps (modulo 2^32).
+     * @return the difference between two RTP timestamps (modulo 2^32).
+     */
+    public static long timestampDiff(long a, long b)
+    {
+        long diff = a - b;
+
+        if (diff < -(1<<31))
```

Comment:
Any reason not to use TimeUtils.rtpDiff? If this needs to stay here, the "1" numerals need to be long to prevent overflows.